### PR TITLE
Fix typo in Http1 documentation

### PR DESCRIPTION
**Pull request by ktec**
Simple typo correction.

### CODE REVIEW

File: lib/raxx.ex [hunk 1]
```diff
@@ -390,10 +390,10 @@ defmodule Raxx do
   ## Examples
 
       iex> request(:GET, "/")
-      ...> |> set_header("referer", "example.com")
```
Comment (CrowdHailer) [inline lib/raxx.ex]: This is not actually incorrect. 
https://en.wikipedia.org/wiki/HTTP_referer

Comment (ktec) [reply lib/raxx.ex]: Wow, learn something new every day! I'll update the PR :+1:

File: lib/raxx/http1.ex [hunk 2]
```diff
@@ -723,7 +723,7 @@ defmodule Raxx.HTTP1 do
     payload_headers =
       case content_length(headers) do
         nil ->
-          # NOTE `:erlang.iolist_size/1` acceps binaries, i.e. should be `:erlang.iodata_size/1`
+          # NOTE `:erlang.iolist_size/1` accepts binaries, i.e. should be `:erlang.iodata_size/1`
```
Comment (CrowdHailer) [inline lib/raxx/http1.ex]: Turns out this was just an incorrect comment ` :erlang.iolist_size("")` works fine.
Thanks for bringing it to my attention. will close this PR and delete the comment.

**Comment by CrowdHailer** [comment]
oh merged on autopilot. nevermind

**Pull request merged**